pkg/transport/queuestorage/file.go

```diff
@@ -12,9 +13,30 @@ import (
 	"scratchdata/pkg/queue"
 )
 
-type FileEvent struct {
-	Key  string
-	Path string
+const (
+	MaxFileSize int64 = 100 * 1024 * 1024 // 100MB
+	MaxRows     int64 = 1_000
+	MaxFileAge        = 1 * time.Hour
+)
+
+type NewFileWriterParam struct {
+	Key    string
+	Path   string
+	Store  filestore.StorageBackend
```

Comment:
It doesn't look like some of these parameters are used, ie, `Store` and `Queue`.